client/asset/btc/btc.go

```diff
@@ -5489,10 +5489,7 @@ out:
 				break out
 			}
 		}
-		blockHash = &blk.Header.PrevBlock
-		if blockHash == nil {
-			return nil, fmt.Errorf("did not find the bond output %v:%d", txHash, vout)
-		}
```

Comment:
This changes the behavior here. The intent is to loop back to genesis if necessary.

Reply:
Oh…blockHash is never nil so that check was invalid.

Reply:
What happens at genesis? I thought block 0 prev hash would be nil as it doesn't exist.

Reply:
Oh it's `chainhash.Hash{}` or all zeros. Can you check for that?

Reply:
Updated.